workshops/urls.py

```diff
@@ -44,4 +44,6 @@
     url(r'^search/?$', views.search, name='search'),
 
     url(r'^export/(?P<name>[\w\.-]+)/?$', views.export, name='export'),
+
+    url(r'^chart/(?P<name>[\w\.-]+)/?$', views.chart, name='chart'),
```

Comment:
You might consider `r'^chart/(?P<name>{workshops,enrolment})/?$'` (my regex might be off) and then the URL dispatcher will handle 404s for unknown charts in both generating URLS in templates & dispatching to the view.